peachjam/models/settings.py

```diff
@@ -104,5 +104,20 @@ def __str__(self):
         return "Settings"
 
 
+class AdminEmail(models.Model):
```

Comment:
I think this is too complicated. We only need the email address. A space-separated list of email addresses on a single CharField on PeachJamSettings is simpler and is sufficient.